src/core/ui/context/login-methods.tsx

```diff
@@ -63,18 +81,84 @@ const LoginMethodsProvider = (
 };
 
 const useLoginMethods = () => {
-  const {
+  const { loginMethods, selectedLoginMethod, setSelectedLoginMethod } =
+    _useLoginMethods();
+  const { excludeLoginMethodTypes, includeLoginMethodTypes, viewOnly } =
+    useSignInContext();
+  const user = useUser();
+
+  const web3 = useMemo(() => {
+    console.log(
+      loginMethods,
+      excludeLoginMethodTypes,
+      includeLoginMethodTypes,
+      user?.loginMethods,
+      viewOnly
+    );
+    if (!viewOnly && user?.loginMethods.includes(LoginMethodType.Web3)) {
+      console.log('user');
```

Comment:
nit: let's remove these console logs in this file

Reply:
Ups my bad! Thanks